Source/WebCore/platform/graphics/gstreamer/eme/WebKitCommonEncryptionDecryptorGStreamer.cpp

```diff
@@ -153,8 +153,10 @@ static GstCaps* transformCaps(GstBaseTransform* base, GstPadDirection direction,
 
             outgoingStructure = GUniquePtr<GstStructure>(gst_structure_copy(incomingStructure));
 
-            if (!canDoPassthrough)
-                gst_structure_set_name(outgoingStructure.get(), gst_structure_get_string(outgoingStructure.get(), "original-media-type"));
+            if (!canDoPassthrough) {
+                if (auto originalMediaType = WebCore::gstStructureGet<ASCIILiteral>(outgoingStructure.get(), "original-media-type"_s))
+                    gst_structure_set_name(outgoingStructure.get(), originalMediaType->characters());
```

Comment:
I think it would be better to not `if` here and just `ASSERT` (or even `RELEASE_ASSERT`) on the result for extra safety.